src/voicegroup.ts

```diff
@@ -1,24 +1,32 @@
 // [VexFlow](http://vexflow.com) - Copyright (c) Mohit Muthanna 2010.
 
+import { ModifierContext } from './modifiercontext';
```

Comment:
since you don't use a constructor of ModifierContext, I think it would be wiser just only import a type:
```typescript
import type { ModifierContext } from './modifiercontext';
```

Same for Voice

Reply:
Good idea.